core_lib/src/tool/basetool.h

```diff
@@ -109,6 +110,7 @@ class BaseTool : public QObject
     virtual void setFeather(const qreal feather);
     virtual void setInvisibility(const bool invisibility);
     virtual void setBezier(const bool bezier_state);
+    virtual void setClosed(const bool closed);
```

Comment:
Maybe we could call this setClosedPath instead so it's more consistent with the other places, that would also make the name less ambiguous.